src/odemis/gui/util/img.py

```diff
@@ -848,16 +848,21 @@ def calculate_ticks(value_range, client_size, orientation, tick_spacing):
         logging.info("Trying to compute legend tick without range")
         return None
 
-    min_val, max_val = value_range[0], value_range[-1]
+    reverse = value_range[0] > value_range[-1]
+    if reverse:
+        min_val, max_val = value_range[-1], value_range[0]
+    else:
+        min_val, max_val = value_range[0], value_range[-1]
 
     # Get the horizontal/vertical space in pixels
     if orientation == wx.HORIZONTAL:
         pixel_space = client_size[0]
         # Don't display ticks too close from the left border
         min_pixel = 10
     else:
+        # Don't display ticks too close from the border
         pixel_space = client_size[1]
-        min_pixel = 0
+        min_pixel = 10
```

Comment:
If in both if and else the min_pixel is set to 10 shoudln't it be set outside the if statement?
Especially since the comment:
>  Don't display ticks too close from the border
Applies to both the horizontal and vertical case.

Reply:
Yes, I first want to be sure that 'Eric is ok with defining min_pixel for both vertical and horizontal and I will merge it.